package.json

```diff
@@ -1,55 +1,103 @@
 {
-  "name": "react-hook-form-error",
-  "version": "1.0.2",
+  "name": "@hookform/error-message",
+  "version": "1.0.0",
   "description": "React Hook Form error message component",
-  "repository": "git@github.com:react-hook-form/react-hook-form-error.git",
-  "author": "bluebill1049 <bluebill1049@hotmail.com>",
-  "license": "MIT",
   "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "module": "dist/index.esm.js",
+  "umd:main": "dist/index.umd.min.js",
+  "unpkg": "dist/index.umd.min.js",
+  "jsdelivr": "dist/index.umd.min.js",
+  "jsnext:main": "index.esm.js",
   "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
   "scripts": {
-    "test": "jest --runInBand",
-    "test:watch": "yarn test -- --watchAll",
-    "clean": "rm -rf dist",
-    "build": "rollup -c",
-    "prettier": "prettier --write './src/**/*.ts' './src/**/*.tsx' --config ./.prettierrc",
-    "lint": "yarn lint:check --fix && yarn prettier",
-    "lint:check": "eslint ./src --ext .tsx,.ts --ignore-pattern *.test.ts --report-unused-disable-directives",
-    "postrelease": "yarn publish && git push --follow-tags",
-    "prepublishOnly": "yarn lint && yarn test && yarn run clean && yarn build"
+    "build": "npm run clean && rollup -c",
+    "clean": "rimraf dist",
+    "lint": "eslint '**/*.{js,ts,tsx}'",
+    "lint:fix": "npm run lint -- --fix",
+    "lint:types": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage"
   },
+  "keywords": [
+    "react",
+    "react-component",
+    "form",
+    "forms",
+    "form-validation",
+    "validation",
+    "hookform",
+    "react-hook-form",
+    "typescript"
+  ],
```

Comment:
@bluebill1049 What do you think this keywords?

Reply:
looks great.